openai.embeddings/Package.md

```diff
@@ -1 +1,67 @@
-# Client to connect to OpenAI
+Connects to the OpenAI Embeddings API from Ballerina with `ballerinax/openai.embeddings` package.
+
+## Package Overview
+The `ballerinax/openai.embeddings` package is a [Ballerina](https://ballerina.io/) connector for OpenAI's Embeddings API. The connector provides access to state-of-the-art models for embeddings. By using the `ballerinax/openai.embeddings` package, you can easily integrate OpenAI's advanced AI capabilities into your own Ballerina applications.
+
+|                             | Version         |
+|-----------------------------|-----------------|
+| OpenAI API                  | 1.2.0           |
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+* Create an [OpenAI Account](https://beta.openai.com/signup/).
+* Obtain an API key by following [these instructions](https://platform.openai.com/docs/api-reference/authentication).
+
+## Quickstart
+
+To use the OpenAI Embeddings connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import connector
+First, import the `ballerinax/openai.embeddings` module into the Ballerina project.
+
+```ballerina
+import ballerinax/openai.embeddings;
+```
+
+### Step 2: Create a new connector instance
+Create and initialize `embeddings:Client` with the `apiKey` obtained.
+```ballerina
+    embeddings:Client embeddingsClient = check new ({
+        auth: {
+            token: "sk-XXXXXXXXX"
+        }
+    });
+```
+
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
+
+    Following is an example on obtaining embeddings from GPT-3 ada model:
+
+    Obtain embeddings from ada model
```

Comment:
```suggestion
    Obtain embeddings from ada model
```
Is this a sub-heading? What is `ada` here?

Reply:
https://github.com/hevayo/ballerina-openai-client/pull/10/files#r1138523502